modules/unsupported/arcsde/common/pom.xml

```diff
@@ -123,7 +123,7 @@
       </build>
     </profile>
     <profile>
-      <id>arcsde</id>
+      <id>arcsde-jars</id>
```

Comment:
What happened here?

Reply:
not sure, let me double check.

Reply:
ok, that's actually a fix. That profile is meant for development when you do have the proprietary jars in your local maven repo. Both common/pom.xml and datastore/pom.xml use it, but in common it was called `arcsde` and in datastore `arcsde-jars`.